package.json

```diff
@@ -9,6 +9,7 @@
     "run-ios": "yarn install:pod && react-native run-ios",
     "preinstall": "node -e \"if(process.env.npm_execpath.indexOf('yarn') === -1) throw new Error('You must use Yarn to install, not NPM')\"",
     "install:pod": "cd ios && bundle install && bundle exec pod install",
+    "install:all": "find . -type dir -name node_modules | xargs rm -rf && yarn",
```

Comment:
deleting the `node_modules/` will invalidate the caches which speed up the workflows... ideally we can avoid this.

Reply:
`find . -type dir -name node_modules | yarn` 🤔